tools/ctl/ctl.c

```diff
@@ -135,7 +141,12 @@ static int read_setup(struct ctl_data *ctl_data)
 
 		n++;
 	}
+
 	fprintf(stdout, "\n");
+	if (ctl_data->no_abi) {
+		hdr->size = n * sizeof(int);
+		n += abi_size;
+	}
```

Comment:
@singalsu since this is common for both binary and CSV txt, can we move this to read_done, then we can save one in the "if (ctl_data->binary) {}" case?